client/scss/overrides/_vendor.tippy.scss

```diff
@@ -31,9 +31,17 @@
   }
 }
 
+.tippy-box[data-theme='dropdown-button'] {
```

Comment:
e.g. in this case I guess I kind of understand why this is here and not in `dropdown-button`, but then in `dropdown-button` we also have `[data-tippy-root]`